src/libsync/discoveryphase.cpp

```diff
@@ -305,7 +306,13 @@ void DiscoverySingleLocalDirectoryJob::run() {
         i.inode = dirent->inode;
         i.isDirectory = dirent->type == ItemTypeDirectory;
         i.isHidden = dirent->is_hidden;
-        i.isSymLink = dirent->type == ItemTypeSoftLink;
+#ifdef Q_OS_WIN
```

Comment:
Why isn't that one done in csync_vio_local_win.cpp instead? With what you're trying to achieve, this would be a more natural fit to me to not set ItemTypeSoftLink there in the first place.

Reply:
@er-vin In fact, I am fine with **ItemTypeSoftLink**, but, I've moved this into **csync_vio_local_win.cpp** indeed. Pushed a new commit with changes.